dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -136,6 +136,9 @@ def clean_row(cls, row, long_to_short_dict, fields):
             if value is not None:
                 # Remove extra whitespace
                 value = value.strip()
+                # If field is string and has triple quotes, remove. Single quote because csv converts triple to single
+                if field_type in ["STRING"] and value.startswith('"') and value.endswith('"'):
+                        value = value[1:-1]
```

Comment:
Just wondering, do we want to strip() again here?

Also, just making sure, does this work for any field? There's some chance that they'll wrap all fields in quotes and if it's an integer field we want to strip them anyway. There's no situation in which we wouldn't want to strip the surrounding quotes.